api4/bot.go

```diff
@@ -39,7 +39,7 @@ func createBot(c *Context, w http.ResponseWriter, r *http.Request) {
 
 	auditRec := c.MakeAuditRecord("createBot", audit.Fail)
 	defer c.LogAuditRec(auditRec)
-	auditRec.AddMeta("bot", bot)
+	auditRec.AddEventParameter("bot", bot)
```

Comment:
```suggestion
	auditRec.AddEventParameter("bot", bot.Auditable())
```

We should call `Auditable` on the struct one way or another. Either explicitly as above, or we could alternatively try to do a type cast to the interface in the `AddEventParameter` function and then call the function, that way the developer cannot can forget it. @agnivade may have thoughts on this as well.